site/app/models/RainbowCustomization.php

```diff
@@ -537,6 +537,14 @@ public function getPlagiarism(): array {
         return !is_null($this->RCJSON) ? $this->RCJSON->getPlagiarism() : [];
     }
 
+    /**
+     * Get manual grades from json file if there are any
+     *
+     * @return array<object>  array of manual grades JSON object
+     */
+    public function getManualGrades(): array {
+        return !is_null($this->RCJSON) ? $this->RCJSON->getManualGrades() : [];
```

Comment:
This is a great opportunity to us PHP's null coalescing operator.  I believe the null coalescing operator eliminates the need to have a nullsafe operator.

```suggestion
        return $this->RCJSON->getManualGrades() ?? [];
```

Reply:
[f0bbef1](https://github.com/Submitty/Submitty/pull/10653/commits/f0bbef17b04826d8fc110beea691e5546e418057)

Reply:
Need to use null-safe operator [59a1948](https://github.com/Submitty/Submitty/pull/10653/commits/59a19487b713b67a0fdbbfda2bcc8a04a6151bd2)